packages/compass-settings/src/components/settings/gen-ai-settings.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SettingsList from './settings-list';
+import { useIsAIFeatureEnabled } from 'compass-preferences-model/provider';
+import { ConnectedAtlasLoginSettings } from './atlas-login';
+import { css, spacing } from '@mongodb-js/compass-components';
+
+const atlasSettingsContainerStyles = css({
+  marginTop: spacing[3],
+});
+
+export const GenAISettings: React.FunctionComponent = () => {
+  const aiFeatureEnabled = useIsAIFeatureEnabled();
+
+  return (
+    <div data-testid="gen-ai-settings">
+      <div>
+        Compass users with MongoDB Atlas accounts enjoy access to an extended
+        set of generative AI functionality, starting with natural language
+        processing for quicker query and aggregation authoring.
+      </div>
+      <SettingsList fields={['enableGenAIFeatures']} />
+
+      {aiFeatureEnabled && (
```

Comment:
Should we check the setting sandbox value here? Then it'll show/hide the login depending.

Reply:
There is a [follow-up ticket](https://jira.mongodb.org/browse/COMPASS-7800) to adjust this behavior. After chatting with the design team, we are not sure if we want to login before a user has saved the `enableGenAIFeatures` setting.